yellowbrick/features/pcoords.py

```diff
@@ -56,14 +59,20 @@ def parallel_coordinates(X, y=None, ax=None, features=None, classes=None,
     classes : list of strings, default: None
         The names of the classes in the target
 
+    normalize : string or None, default: None
+        specifies which normalization method to use, if any
+
+    sample : float or int, default: 1.0
+        specifies how many examples to display from the data
+
```

Comment:
Hmm, I didn't realize that was a convention - perhaps I just put in docstring stubs and meant to update them later? Needless to say, I'd appreciate if you made the match, but don't let it block the merge.